tox.ini

```diff
@@ -0,0 +1,66 @@
+[tox]
+envlist = black, flake8, pylint, yamllint, py36, py38
+skip_missing_interpreters = True
+
+[testenv:black]
+skip_install = true
+basepython = python3.6
+changedir = {toxinidir}
+deps =
+    black==21.6b0
+# style configured via pyproject.toml
+commands =
+    black \
+        --check \
+        --diff \
+        {posargs} \
+        ./
+
+[testenv:flake8]
+basepython = python3.6
+skip_install = true
+changedir = {toxinidir}
+deps =
+    flake8==3.7.9
+commands =
+    flake8 \
+        --statistics {posargs} \
+        main.py \
+        config.py \
+        githublib.py \
+        jiralib.py \
+        ticket.py \
+        tests/test_github.py \
+        tests/test_config.py
```

Comment:
Why do you need to specify specific files and not the whole folders? This is not maintainable.
Same for the other occurrence of this.

Reply:
pylint has problems with it (at least on CI)
there is even an open issue about it 
https://github.com/PyCQA/pylint/issues/352